pyproject.toml

```diff
@@ -3,15 +3,15 @@
 
 [build-system]
 requires = [
-  "setuptools == 67.7.2",
+  "setuptools == 68.1.0",
   "setuptools_scm[toml] == 7.1.0",
-  "frequenz-repo-config[api] == 0.4.0",
+  "frequenz-repo-config[api] == 0.6.2",
 ]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "frequenz-api-weather"
-description = "Frequenz gRPC API for retrieving weather forecasts."
+description = "Protobuf spec for the weather API."
```

Comment:
I think this should stay as it was. Maybe you need to update the `cookiecutter-replay.json` file.